ListView_GetTopIndex can return negative values, GetSubItemRect
ought not to choke on it.
---
 src/msw/listctrl.cpp |    4 ----
 1 file changed, 4 deletions(-)

Index: wxWidgets-3.0.2/src/msw/listctrl.cpp
===================================================================
--- wxWidgets-3.0.2.orig/src/msw/listctrl.cpp
+++ wxWidgets-3.0.2/src/msw/listctrl.cpp
@@ -1077,10 +1077,6 @@ bool wxListCtrl::GetSubItemRect(long ite
                     (subItem >= 0 && subItem < GetColumnCount()),
                  false, wxT("invalid sub item index") );
 
-    // use wxCHECK_MSG against "item" too, for coherency with the generic implementation:
-    wxCHECK_MSG( item >= 0 && item < GetItemCount(), false,
-                 wxT("invalid item in GetSubItemRect") );
-
     int codeWin;
     if ( code == wxLIST_RECT_BOUNDS )
         codeWin = LVIR_BOUNDS;
